core/core/src/sf_core/map_std_impl/mod.rs

```diff
@@ -90,11 +122,36 @@ impl MapStdUnstable for MapStdImpl {
             None => Err(MapHttpCallHeadError::InvalidHandle),
             Some(request) => {
                 let response = request.into_response()?;
+                let status = response.status();
+                let headers = response.headers().clone();
+                let body = response.into_body();
+
+                // IDEA: mark this branch as unlikely?
```

Comment:
I don't understand this comment.

Reply:
I would like to give the compiler a hint that the branch is unlikely to be taken. This should help the compiler reorder the code more optimally, but I didn't want to spend too much on it since there is no stable counterpart to the intrinsic (https://doc.rust-lang.org/std/intrinsics/fn.unlikely.html). Interesting thread also [here](https://users.rust-lang.org/t/compiler-hint-for-unlikely-likely-for-if-branches/62102/13) and some advanced reading [here](https://llvm.org/docs/BranchWeightMetadata.html).

But to be honest I don't know how this interacts with WASM at all, so maybe it's entirely pointless.